docs/pages/access-controls/guides/impersonation.mdx

```diff
@@ -312,23 +312,16 @@ $ tsh login --proxy=teleport.example.com --user=alice --auth=local
 $ tctl auth sign --user=security-scanner --format=openssh --out=security-scanner --ttl=10h
 ```
 
-Here is a summary of variables and functions we used in this guide:
-
-<table>
-<tr>
-  <th>Variable or Function</th>
-  <th>Description</th>
-</tr>
-<tr>
-  <td>`user.spec.traits`</td>
-  <td>Access traits of local or SSO user.</td>
-</tr>
-<tr>
-  <td>`contains(list, var)`</td>
-  <td>Checks whether list contains variable</td>
-</tr>
-<tr>
-  <td>`equals(var, var)`</td>
-  <td>Checks whether one variable is equal another</td>
-</tr>
-</table>
+### Filter fields explained
+
+Explanation of the fields used in this guide for the `where` conditions.
```

Comment:
```suggestion
Here is an explanation of the fields used in the `where` conditions within this guide.
```
Suggested making this a complete sentence to create a slightly more conversational tone.